src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
Apparently you're not only one to not expect this @nicolas-grekas ^^
As you can see in the changes log, ``resolveValue`` is wrongly used in many places.